source/include/ota.h

```diff
@@ -62,9 +62,8 @@
  *
  */
 
-/* MISRA rule 8.6 requires identifier with external linkage to have exact one external definition.
- * However, this variable is defined in OTA platform abstraction layer implementation, which is
- * not in this repository but in C-SDK and amazon-freertos repo, so it's a false positive. */
+/* MISRA Ref 8.6.1 [External linkage] */
```

Comment:
Rule 8.6 is disabled in misra.config. Do we need suppression still ?

Reply:
This is actually the only flag of 8.6 if the rule is removed from the config. I've removed the rule from the config and left the suppression in here.